<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTEST">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuSetPicture">
     <property name="title">
      <string>SetPicture</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="actionCross_vs_Zero"/>
     <addaction name="actionApple_vs_Orange"/>
    </widget>
    <addaction name="actionPlay_again"/>
    <addaction name="actionPlay_with_bot"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionQuit"/>
    <addaction name="menuSetPicture"/>
   </widget>
   <addaction name="menuTEST"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPlay_again">
   <property name="text">
    <string>Player vs Player</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPlay_with_bot">
   <property name="text">
    <string>Player vs Bot</string>
   </property>
  </action>
  <action name="actionPlay_without_bot_default">
   <property name="text">
    <string>Play without bot (default)</string>
   </property>
  </action>
  <action name="actionCross_vs_Zero">
   <property name="text">
    <string>Cross vs Zero</string>
   </property>
  </action>
  <action name="actionApple_vs_Orange">
   <property name="text">
    <string>Apple vs Orange</string>
   </property>
  </action>
  <action name="actionasd">
   <property name="text">
    <string>fdsadf</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
